src/theme/border/theme-helper.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from "styled-components";
+import { StyledComponentProps, getCollection, getValue } from "../helpers";
+import { BorderScheme, ThemeCollectionSpec } from "../types";
+import { BorderCollectionMap, BorderSet, StyleProps } from "./types";
+import { LifeSgBorderSet } from "./specs/lifesg-border-set";
+
+const BorderSpec: ThemeCollectionSpec<BorderCollectionMap, BorderScheme> = {
+    collections: {
+        lifesg: LifeSgBorderSet,
+    },
+    defaultValue: "lifesg",
+};
+
+export const dashedBorderStyle =
+    (
+        thickness: number | ((props: any) => string),
```

Comment:
we can enforce thickness to be returned as a pixel number value. this way, we don't have to parse the resolved thickness

```suggestion
        thickness: number | ((props: any) => number),
```